src/app/policies/views/PolicyView.vue

```diff
@@ -1,172 +1,86 @@
 <template>
```

Comment:
**Suggestion**: Can you rename this file to `PolicyDefailView.vue`, please?

Currently, the git diff doesn’t show this file as new because it thinks its the previous `PolicyView.vue` file. I hope changing the name allows the diff to associate old `PolicyView.vue` to new `PolicyListView.vue` and see `PolicyDetailView.vue` as an addition.

Reply:
Oh gotcha yeah one sec

Reply:
Changed that up, I spotted another bit of renaming in the tests also so added that on the end here